### PR TITLE
add join table for visits

### PR DESCRIPTION
**Pull request by mosidrum**
add join table for visits and queried data to answer some questions

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -65,3 +65,45 @@ WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
...
+--How many visits were with a vet that did not specialize in that animal's species?
+  SELECT COUNT(*) FROM visits v JOIN animals a ON a.id = v.animal_id
+  JOIN vets vs ON vs.id = v.vets_id LEFT JOIN specializations spec ON
+  spec.vets_id = vs.id AND spec.species_id = a.species_id WHERE spec.vets_id IS NULL;
+
+--What specialty should Maisy Smith consider getting? Look for the species she gets the most.
+  SELECT COUNT(visits.animal_id) FROM visits where vets_id =
+ (select id from vets WHERE name = 'Maisy Smith') AND
+ animal_id IN (SELECT id from animals WHERE species_id =
+  (SELECT id FROM species WHERE name = 'Digimon'));
```
Comment (LionRouge1) [inline queries.sql]: - [ ] Please note that for this question (**What specialty should Maisy Smith consider getting? Look for the species she gets the most.**), we are looking for the species that **Maisy Smith** get the most not the number of times she gets 'Digimon'. With that in mind, I think your query could look like this :point_down: 
```sql
  SELECT species.name, count(*)
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN species ON animals.species_id = species.id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE vet.name = 'Maisy Smith'
  GROUP BY species.name
```

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @mosidrum, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job using `JOIN` for many-to-many relationships :1st_place_medal: 
- Nice work providing SQL queries to answer the questions :+1: 
- Add the queries' result screenshots to PR :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @mosidrum , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**